lib/pages/discover.dart

```diff
@@ -33,16 +34,16 @@ class _DiscoverDevicesState extends State<DiscoverDevices> {
           deviceList.add(addr.ip);
         });
       }
-    });
+    }).onDone(_refreshController.refreshCompleted);
   }
 
   @override
   void initState() {
```

Comment:
As you also mentioned before, it'll be fine to remove `initState()`.

Reply:
Alright:+1:

Reply:
@sanket143  done.